sysutils/apuled/src/opnsense/scripts/apuled/apuledctl

```diff
@@ -1,32 +1,34 @@
 #!/bin/sh
+#    Copyright (c) 2019 Cloudfence - Julio Camargo (JCC)
```

Comment:
this is also off. second line empty "#", third line down indent is "# " (one space only).

Reply:
Done!